documentation/book/cluster-operator.adoc

```diff
@@ -1213,6 +1219,10 @@ of the Kafka cluster ConfigMap.
 The image name to use as a default when deploying Zookeeper, if
 no image is specified as the `zookeeper-image` in the <<kafka_config_map_details,Kafka cluster ConfigMap>>.
 
+[[STRIMZI_DEFAULT_TLS_SIDECAR_IMAGE]] `STRIMZI_DEFAULT_TLS_SIDECAR_IMAGE`:: Optional, default `strimzi/zookeeper-stunnel:latest`.
```

Comment:
There is a mistake here, it should be `STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE`. can you address this @kyguy please ?